sql/moz-fx-data-shared-prod/mozilla_vpn_derived/fxa_attribution_v1/query.sql

```diff
@@ -51,18 +51,41 @@ flows AS (
     fxa_content_auth_stdout_events
   WHERE
     IF(@date IS NULL, partition_date < CURRENT_DATE, partition_date = @date)
-    -- cannot filter service because
-    AND (
+    AND flow_id IS NOT NULL
+  GROUP BY
+    flow_id
+  HAVING
+    LOGICAL_OR(
```

Comment:
> If I am not mistaken, would the multi touch approach resolve non-VPN attribution being selected?

This is still doing first touch attribution, so no change in that respect.

However, after further testing I think my previous assertion that this "_definitely broadens the pool of potential attribution parameters and may result in non-VPN attribution parameters being selected here if they came first_" is incorrect, and these changes will actually result in _fewer_ non-VPN attributions being selected:
- Flows started from www.mozilla.org FxA button views aren't persistent across www.mozilla.org page views like I originally thought.  I had assumed once a flow ID was assigned it would do something like set a cookie so that the same flow ID would be reused across www.mozilla.org page views within a limited time period, but it appears that each www.mozilla.org page view with FxA links generates a new flow ID for the FxA links on that page, and more events with that flow ID only occur if they click the FxA link.
- Previous logic was including all `fxa_rp_button - view` events with null `service`, but the vast majority of those events were for the "Join Firefox" button on the Firefox homepage and various FxA product links on the Firefox "What's New" pages.  The new logic won't include those non-VPN flows.